src/states_screens/tracks_screen.cpp

```diff
@@ -152,6 +152,8 @@ void TracksScreen::beforeAddingWidget()
     //I18N: track group name
     FOR_GETTEXT_ONLY( _("standard") )
     //I18N: track group name
+    FOR_GETTEXT_ONLY( _("nextgen") )
+    //I18N: track group name
```

Comment:
Sorry for the delay before reviewing this... This string should not be translatable, it never appears in the final game